### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by njons**
- [x] Cleaned up component folder structure (further splitting needed)
screen shot 2018-10-14 at 17 34 30

- [x] Fixed layout differences at the bottom of pages
stations
cases
revision

- [x] Made all pages accessible via the navbar render the same content (useful for planning how to split the first pages into dynamic content that can render in the different sections in case we get to that)
screen shot 2018-10-14 at 17 40 36

- [x] created an [anatomy Figma](https://www.figma.com/file/mJ1U0KSnzaKvA1fteKQYDM/OSCE-Bosskey-testing-colors?node-id=159%3A4) layout as a cheat sheet for the component names
screen shot 2018-10-14 at 17 41 13

- [x] tests passing :heart: Travis

### CODE REVIEW

**Comment by njons** [comment]
Hey @Brymon, @sima-qian, @martingaston - I didn't realise that we/I left a PR waiting on Friday. 😱  

Once it's merged, I'll pull the new master into this PR and let you know when its fully ready to review (meanwhile do feel free to wrap your head around the stuff I have listed in the description).

File: index.html [hunk 1]
```diff
@@ -5,12 +5,12 @@
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>OSCEBossKey</title>
-    <link href="https://fonts.googleapis.com/css?family=Nova+Round|Roboto:400,900" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Nova+Round|Roboto:400,700,900" rel="stylesheet">
```
Comment (sima-qian) [inline index.html]: Do we know if loading four fonts will have a noticeable effect on initial loading time?

**Comment by sima-qian** [comment]
Making a note about merging the `History`, `Extras` and `Examinations` components -- see #115

File: src/components/Pages/Revision/RevisionPage.js [hunk 2]
```diff
@@ -21,14 +21,14 @@ export default class HistoryCaseRevision extends React.Component {
 
   componentDidMount() {
     const { station, caseid } = this.props.match.params;
-    const markScheme = dummyData.history[station][caseid]["mark-scheme"].map(
-      element => ({ text: element, completed: false })
+    const revisionList = dummyData.history[station][caseid]["mark-scheme"].map(
```
Comment (sima-qian) [inline src/components/Pages/Revision/RevisionPage.js]: Given all the amazing work you've done for this PR, I'm going to be a complete cow and say that I don't like this variable name. Not sure what was wrong with `markScheme` -- if we want to be clear it's a list, we could go with `markSchemeList`?

File: src/components/Stations/AddNew.js [hunk 3]
```diff
@@ -3,11 +3,11 @@
 import React from "react";
 import { Link } from "react-router-dom";
 
-export default class AddNewStation extends React.Component {
+export default class AddNew extends React.Component {
```
Comment (sima-qian) [inline src/components/Stations/AddNew.js]: Again, I'm being pathetically pedantic. (Sorry.) I think having two components called `AddNew`, even though they're in separate folders, could be confusing and perhaps isn't best practice when they are quite different things. Would it be possible to go back to `AddNewCase` and `AddNewStation`?

Comment (njons) [reply src/components/Stations/AddNew.js]: I think there are a few things going on here: 
1. Naming of the variables and components in the Case and Station folders specifically is till ongoing. The words "Stations" and "Cases" were confusing to me on Friday (and I assume would be to any future dev) so I would suggest exchanging these to "Tiles" and "Banners" which holds information not only of what the elements are but also how they look 

2. The duplicate naming in these folders came from me starting to identify opportunities for refactoring (so see this as an attempt of an suggestion) and the naming does not have to stay duplicate, but i'd like to discuss refactoring at some point 

3. To answer your question we can call them "AddNewTile" and "AddNewBanner", but I would like to open up the discussion to possibly refactor these (it depends a bit on what the back looks like)

4. finally, any set names should be captured in the anatomy figma thingy (I suggest we stick an image of it in the wiki when we hand over)

**Pull request merged**